src/components/z-book-cover/styles.css

```diff
@@ -0,0 +1,68 @@
+:host,
+* {
+  box-sizing: border-box;
+}
+
+:host {
+  --z-book-cover-width: 100%;
+  --z-book-cover-height: 378px;
+  --z-book-cover-border-radius: var(--border-radius);
+  --z-book-cover-stack-offset-x: 8px;
+  --z-book-cover-stack-offset-y: 12px;
+  --z-book-cover-vertical-alignment: start;
+  --z-book-cover-horizontal-alignment: start;
```

Comment:
Have you considered setting the default to "center" both for horizontal and vertical? Is there a reason why you preferred start ?

Reply:
Yes, the reason is that when the image has a different aspect ratio than the container, the ratio of both the image and the container is preserved and some white space can appear. When this happens, the image has to be put at the end vertically (actually this `start` is an error, I will fix it) and at `start` horizontally by design (I took the z-book-card defaults as a reference).

Reply:
Direi che questo dovrebbe essere il Figma da seguire: https://www.figma.com/design/4XbIVG4RAt1xRrY4mi6KBT/Catalogo-e-Shop-pt.1?node-id=14998-30786&m=dev